advanced/how_to_debug.rst

```diff
@@ -0,0 +1,101 @@
+.. _how_to_debug:
+
+How to debug
+============
+
+Console and debugger
+--------------------
+
+When you test your game made with Escoria, you can get some errors and warnings
+in the console. These information can be very useful to debug the issue.
+
+If Escoria encounters an error or a crash, it will stop the execution of the
+game and return to the editor, showing you the line that broke. Usually, this
+line will be located in Escoria's logger class, as this class is in charge of
+showing the errors before stopping the execution.
+
+Godot also shows the debugger panel, but it is advised to check the output log
+first.
+
+.. image:: img/crash-error-console.png
+    :align: center
+    :alt: Crash error: console
+
+The console shows many lines, depending on the log level defined in Project
+Settings (from finest to highest-level: TRACE, DEBUG, INFO, WARNING, ERROR).
+
+In the example above, the error is explained in the line starting with ``(E)``
+("E" for "Error", "D" for "Debug", "I" for "Info", "T" for "Trace"). The reason
+of this error is that Escoria attempted to load an inexisting file. The (E)rror
+line provides the path to the file that was expected. At this point, fixing the
+issue is easy: either create the missing file, or remove the references to this
+inexisting file in the scenes that have it.
+
+Of course, this example is straightforward. Sometimes, the issue can be
```

Comment:
This example is straight forward, of course.

Reply:
Fixed